elements/search/document_picker/nuxeo-document_picker-search-form.html

```diff
@@ -0,0 +1,43 @@
+<!--
+`nuxeo-document_picker-search-form`
+@group Nuxeo UI
+@element nuxeo-document_picker-search-form
+-->
+<dom-module id="nuxeo-document_picker-search-form">
+  <template>
+    <style include="nuxeo-styles"></style>
+    <nuxeo-input
```

Comment:
Since this is a new element, we might want to add here the `aria-label` too. What do you think @semisse? Should we do it for the search form and search results?

Reply:
Well noted, Ricardo. In this case we should add:
```suggestion
    on-keydown="_checkForEnter"
    aria-label$="[[i18n('pickerSearch.title')]]"
    ></nuxeo-input>
```